docs/registry.md

```diff
@@ -67,9 +69,11 @@ One can set them to 1 and 1 by default to always unfreeze all tokens.
 
 ### Decision lambda
 
-It simply applies all updates from the accepted proposal one by one.
-They are applied in the same order as specified in the proposal, the head of the
-list is applied first.
+For the Transfer proposal, It simply applies all updates from the accepted
```

Comment:
```suggestion
Except for the Guardian updates, It simply applies all updates from the accepted
```
I think the previous text didn't mention transfer proposals specifically as the rest of the comment is true for all the update proposals (except the new guardian one).